### DISCUSSION TITLE
Buggy swipe gesture to change months

### DISCUSSION THREAD

**Discussion by rafnov** [FossifyOrg/General-Discussion#463] [General]
I got this when trying to switch months with swipe:
https://github.com/user-attachments/assets/e98653cb-08fc-4ac7-9913-772da5da6adb

Feels like the screen looses its touch and goes back to previous month, not the one I'm trying to switch to. But it's not the screen, see the touch dot. 
I am  able to switch months with arrows, so only switching by swiping looks buggy.
I'm on stock Android 12, Realme 7 Pro.

**Comment by Aga-C** [comment]
It's already reported here: https://github.com/FossifyOrg/Calendar/issues/153.
